FAQ.md

```diff
@@ -0,0 +1,24 @@
+## Frequently Asked Questions
+
+- [Should I add an Xcode project to a gitignore file?](#should-i-add-an-xcode-project-to-a-gitignore-file)
+- [What happens when I switch a branch?](#what-happens-when-i-switch-a-branch)
+
+### Should I add an Xcode project to a gitignore file?
```

Comment:
This question is already answered in the [frequently answered questions](https://tuist.io/faq) page on the website so I'd avoid duplications.